docs/book/src/SUMMARY.md

```diff
@@ -62,6 +62,7 @@
         - [Metadata propagation](./developer/architecture/controllers/metadata-propagation.md)
```

Comment:
This is where I mean - move the ClusterResourceSet up above Metadata propagation and below ClusterTopology in the summary

Reply:
Done